src/data/language.js

```diff
@@ -0,0 +1,9 @@
+const languages = {
+    ar : 'Arabic',
+    en : 'English',
+    cn : 'Chinese',
```

Comment:
Chinese is zh, please use https://github.com/scbd/online-reporting-tool/blob/master/app-data/languages.js